.circleci/config.yml

```diff
@@ -90,9 +90,19 @@ jobs:
             source emsdk_env.sh
             python scripts/test.py
 
+  build-docker-image:
+    machine: true
+    steps:
+      - checkout
+      - run: cd docker && make version=${CIRCLE_TAG}-upstream
+      - run: |
```

Comment:
Does this run at the top level, or within the `docker` subdirectory?   (i.e. does the `cd` in the previous run apply to this run too?

Reply:
I am pretty sure that docker push and login do not require the Docker file but use the tag to reference the build docker image stored in the local registry. Right, @trzecieu?

Reply:
That's right, push and login can be done from any place.